sw/device/silicon_creator/rom_ext/doc/si_val.md

```diff
@@ -0,0 +1,59 @@
+# SiVal ROM\_EXT
+
+## Lockdowns
+
+The following lockdowns are provided by the SiVal ROM\_EXT.
+
+### AST
+
+TODO: Lockdown the AST.
+
+### ePMP
+
+The ePMP will allow access to the FLASH, to the MMIO region and to SRAM.
+
+An example of how the SiVal ROM\_EXT will configure the ePMP:
+```
+ 0: 20010400 ----- ---- sz=00000000     ; OWNER code start.
+ 1: 20013d24   TOR LX-R sz=00003924     ; OWNER code end.
+ 2: 00008000 NAPOT L--R sz=00008000     ; ROM data (can be cleared).
+ 3: 20000400 ----- ---- sz=00000000     ; ROM_EXT code start.
+ 4: 20004fa0   TOR LX-R sz=00004ba0     ; ROM_EXT code end.
+ 5: 20000000 NAPOT L--R sz=00100000     ; FLASH read access.
+ 6: 00000000 ----- ---- sz=00000000
+ 7: 00000000 ----- ---- sz=00000000
+ 8: 00000000 ----- ---- sz=00000000
+ 9: 00000000 ----- ---- sz=00000000
+10: 40130000 NAPOT L--- sz=00001000     ; OTP MMIO lockout.
+11: 40000000 NAPOT L-WR sz=10000000     ; MMIO region.
```

Comment:
Another TODO here is to decide on the final ordering as M code in silicon owner stages can use a lower index ePMP entry to override this permission.